<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>28</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnBrowse">
   <property name="geometry">
    <rect>
     <x>1148</x>
     <y>40</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>图像保存路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPath">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>721</width>
     <height>51</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkFPS">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>显示帧频</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkDefog">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>去雾</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkClock">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>显示时间</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>180</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1920x1080</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1280x1024</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1024x768</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>640x480</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>分辨率</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
